docs/SUMMARY.md

```diff
@@ -15,7 +15,10 @@
     * [Debugging techniques](guides/debugging.md)
     * [Multi-platform builds](guides/multi-platform.md)
     * [CI integration](guides/ci-integration.md)
-    * [Pushing and Pulling from AWS ECR](guides/pushing-images-to-ECR.md)
+    * Working With Cloud Providers
+        * [Pushing and Pulling from AWS ECR](guides/pushing-images-to-ECR.md)
+        * [Pushing and Pulling Images with GCP Artifact Registry](guides/pushing-images-to-GCR.md)
+        * [Pushing and Pulling Images with Azure ACR](guides/pushing-images-to-ACR.md)
```

Comment:
Shorter names look better here - how about just:

```suggestion
    * Working With Cloud Providers
        * [AWS ECR](guides/pushing-images-to-ECR.md)
        * [GCP Artifact Registry](guides/pushing-images-to-GCR.md)
        * [Azure ACR](guides/pushing-images-to-ACR.md)
```

Reply:
Check my latest push, I pruned that up

Reply:
But had lengthened the upper one. These guides arent quite wide enough to warrant a "Working With Cloud Providers"